src/api/Jalasoft.TeamUp.Projects.API/Controllers/GetProject.cs

```diff
@@ -0,0 +1,33 @@
+namespace Jalasoft.TeamUp.Projects.API.Controllers
+{
+    using System.Net;
+    using Jalasoft.TeamUp.Projects.Core.Interfaces;
+    using Jalasoft.TeamUp.Projects.Models;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Extensions.Http;
+    using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Attributes;
+    using Microsoft.OpenApi.Models;
+
+    public class GetProject
+    {
+        private readonly IProjectService projectService;
+
+        public GetProject(IProjectService projectService)
+        {
+            this.projectService = projectService;
+        }
+
+        [FunctionName("GetProject")]
+        [OpenApiOperation(operationId: "Run", tags: new[] { "Projects" })]
+        [OpenApiResponseWithBody(statusCode: HttpStatusCode.OK, contentType: "application/json", bodyType: typeof(Project[]), Description = "Successful response")]
+        [OpenApiResponseWithoutBody(statusCode: HttpStatusCode.NotFound, Description = "Resource not found")]
```

Comment:
are we gonna have this scenario? or an empty array will be returned in case no projects are retrieved?

Reply:
I think so because those who started the project left us those examples